posthog-core/src/index.ts

```diff
@@ -63,38 +57,422 @@ export abstract class PostHogCore {
     this.flushAt = options?.flushAt ? Math.max(options?.flushAt, 1) : 20
     this.flushInterval = options?.flushInterval ?? 10000
     this.captureMode = options?.captureMode || 'form'
-    this.sendFeatureFlagEvent = options?.sendFeatureFlagEvent ?? true
-    // If enable is explicitly set to false we override the optout
-    this._optoutOverride = options?.enable === false
+
     this._retryOptions = {
       retryCount: options?.fetchRetryCount ?? 3,
       retryDelay: options?.fetchRetryDelay ?? 3000,
     }
     this.requestTimeout = options?.requestTimeout ?? 10000 // 10 seconds
-    this._sessionExpirationTimeSeconds = options?.sessionExpirationTimeSeconds ?? 1800 // 30 minutes
+  }
 
-    // NOTE: It is important we don't initiate anything in the constructor as some async IO may still be underway on the parent
-    if (options?.preloadFeatureFlags !== false) {
-      safeSetTimeout(() => {
-        this.reloadFeatureFlags()
-      }, 1)
+  protected getCommonEventProperties(): any {
+    return {
+      $lib: this.getLibraryId(),
+      $lib_version: this.getLibraryVersion(),
     }
   }
 
-  protected getCommonEventProperties(): any {
-    const featureFlags = this.getFeatureFlags()
+  on(event: string, cb: (...args: any[]) => void): () => void {
+    return this._events.on(event, cb)
+  }
+
+  debug(enabled: boolean = true): void {
+    this.removeDebugCallback?.()
+
+    if (enabled) {
+      this.removeDebugCallback = this.on('*', (event, payload) => console.log('PostHog Debug', event, payload))
+    }
+  }
+
+  private buildPayload(payload: { distinct_id: string; event: string; properties?: PostHogEventProperties }): any {
+    return {
+      distinct_id: payload.distinct_id,
+      event: payload.event,
+      properties: {
+        ...(payload.properties || {}),
+        ...this.getCommonEventProperties(), // Common PH props
+      },
+    }
+  }
+
+  /***
+   *** TRACKING
+   ***/
+  identifyStateless(distinctId: string, properties?: PostHogEventProperties, options?: PosthogCaptureOptions): this {
+    // This identify doesn't add anything to set, you have to do it yourself!
```

Comment:
I sort of get why this comment is here but I don't fully understand it 😅

Reply:
Will fix